.github/release-drafter.yml

```diff
@@ -0,0 +1,130 @@
+# https://github.com/release-drafter/release-drafter#configuration-options
+
+# The template for the body of the draft release
+template: |
+  $CHANGES
+
+# The template to use for each merged pull request
+change-template: '* $TITLE @$AUTHOR (#$NUMBER)'
+# Characters to escape in $TITLE when inserting into change-template so that they are not interpreted as Markdown format characters
+change-title-escapes: '\<*_&' # You can add #
+
+# The template for the name of the draft release
+name-template: 'v$RESOLVED_VERSION 🌈'
+# The template for the tag of the draft release
+tag-template: 'v$RESOLVED_VERSION'
+
+# # Template used calculating the next version number for the release
+# version-template: '$MAJOR.$MINOR'
+# # Adjust the $RESOLVED_VERSION variable using labels.
+# version-resolver:
+#   # every release is a whole version by default
+#   default: minor
+#   major:
+#     labels:
+#       - 'feature'
```

Comment:
why is there a feature and feat label? I know "feat" is super trendy on github right now, but tbh I'd prefer (1) full spelling (2) one way to do things. Is there some reason like tooling requires feats but not features and its out of our hands?

Reply:
Nope, we can make the tool handle either `feat` or `feature`. I'm a stickler for this kind of thing myself so I get where you're coming from.

Reply:
I have this whole section commented out for now, but theoretically it could help guide our semver versioning based on the labels used on PRs that get merged.